deepchem/models/torch_models/scscore.py

```diff
@@ -0,0 +1,53 @@
+import torch
+import torch.nn as nn
+from deepchem.data import NumpyDataset
+from deepchem.feat import CircularFingerprint
+from deepchem.models.torch_models.torch_model import TorchModel
+from deepchem.models.losses import L1Loss
+
+
+class ScScore(nn.Module):
+    """
+    ScScore paper - https://pubs.acs.org/doi/abs/10.1021/acs.jcim.7b00622
+
+    ScScore is a FeedForward neural network with 5 hidden layers. Each hidden layer
+    contains 300 neurons. ReLU is applied for all the hidden layers. Sigmoid is applied
+    for the output layer. In the original paper, they didn't use Dropout. But in our
+    implementation, you can set the dropout for all the hidden layers at once.
+    """
+    
+    def __init__(self, n_features: int, layer_sizes: list, dropout: int, **kwargs) -> None:
+        """
+        Arguments
+        """
+        super().__init__()
+        self.dropout = dropout
+        self.input_layer = nn.Linear(n_features, layer_sizes[0])
+        self.hidden_layers = nn.ModuleList()
+        for layer_size in layer_sizes[1:]:
+            self.hidden_layers.append(nn.Linear(layer_size, layer_size))
+        self.output_layer = nn.Linear(layer_sizes[-1],1)
+        
+    def forward(self,x: torch.Tensor) -> torch.Tensor:
+        x = nn.ReLU()(self.input_layer(x))
+        for hidden_layer in self.hidden_layers:
+            x = hidden_layer(x)
+            if self.dropout > 0:
+                x = nn.Dropout(p=self.dropout)(x)
+            x = nn.ReLU()(x)
+        output = self.output_layer(x)
+        output = nn.Sigmoid()(output)
+        return output
+        
+        
+class ScScoreModel(TorchModel):
+    """
```

Comment:
Add docstrings and include model in `docs/source/api_reference/models.rst`

Reply:
I'll add at the end.